src/Core/Translation/Builder/TranslationCatalogueBuilder.php

```diff
@@ -190,6 +190,7 @@ public function getRawCatalogue(
             foreach ($messages as $translationKey => $translationValue) {
                 $translationKey = (string) $translationKey;
                 $message = new Message($translationKey);
+                $message->setMetadata($defaultCatalogue->getMetadata($translationKey, $domainName) ?? []);
```

Comment:
You should update https://github.com/PrestaShop/PrestaShop/blob/b81b9dda9688598ff6199a96615ea5552fc2294e/tests/Unit/Core/Translation/Builder/TranslationCatalogueBuilderTest.php